automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -28,9 +29,46 @@ main() {
     echo "Deploy KubeVirt latest stable release"
     ./hack/deploy-kubevirt.sh
 
+    echo "Get skip tests regex"
+    SKIPPED_TESTS=$(__get_skipped_tests)
+
     # Run KubeMacPool functional tests
     cd ${TMP_COMPONENT_PATH}
-    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" make functest
+
+		echo SKIPPED_TESTS=$SKIPPED_TESTS
+    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml --ginkgo.skip $SKIPPED_TESTS" make functest
+}
+
+function __get_skipped_tests() {
+	local skipped_regex=""
+
+	if [ "$(__is_opt_out)" == "true" ]; then
+		skipped_regex="${skipped_regex} \(opt-in\smode\)"
+	elif [ "$(__is_opt_in)" == "true" ]; then
+		skipped_regex="${skipped_regex} \(opt-out\smode\)"
+	fi
```

Comment:
skipped_regex is local and we have `if/elif` does not make sense to append her.

Reply:
it's intentional, the idea was to create a general function `__get_skipped_tests` that adds regexes to skip. Currently it only adds the opt-mode but maybe later we'll want to concatenate others.
but if you prefer I can remove this part

Reply:
Fair enough